main.py

```diff
@@ -264,7 +265,7 @@ def get_board(self, access_token_in):
                                         requests.get(
                                             msg["data"]["name"],
                                             stream=True,
-                                            proxies=proxy.get_random_proxy(self),
+                                            proxies=proxy.get_random_proxy(self, None),
```

Comment:
Don't pass a `None` value if you don't intend to give any as first argument. If you intend to make it apparent that the `name` is `None` then do the following:
```python
proxies=proxy.get_random_proxy(self, name=None)
```